### PR TITLE
Restrict: global ignore management link to PRO users and fix button label span closure

### PR DESCRIPTION
**Pull request by SteveJonesDev**
This pull request introduces a conditional check for the global ignore management link, ensuring it is only shown to users with the Pro version enabled. It also fixes a minor HTML markup issue in the ignore button.

Permission and feature gating:

* The "Manage Globally Ignored" link is now only displayed if both `$ignore_global` is true and the user has the Pro version (`edac_is_pro()`), restricting global ignore management to Pro users.

Markup correction:

* Fixed the HTML for the ignore button by closing the `<span>` element properly, preventing potential markup issues in the UI.

Fixes: #1308

## Summary by CodeRabbit

* **Bug Fixes**
  * Fixed button markup and data-attribute rendering so controls display and behave properly.

* **Updates**
  * "Manage Globally Ignored" link now appears only when global ignore is enabled and the account is eligible, and its label is localized.

* **Chores**
  * Improved CI linting flow with a fallback step to better handle parse-error checks.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: admin/class-ajax.php [hunk 1]
```diff
@@ -540,10 +540,10 @@ function ( $a, $b ) {
 						$html .= ( true === $ignore_permission || ! empty( $ignore_comment ) ) ? '<label for="edac-details-rule-records-record-ignore-comment-' . $id . '">Comment</label><br>' : '';
 						$html .= ( true === $ignore_permission || ! empty( $ignore_comment ) ) ? '<textarea rows="4" class="edac-details-rule-records-record-ignore-comment" id="edac-details-rule-records-record-ignore-comment-' . $id . '" ' . $ignore_comment_disabled . '>' . $ignore_comment . '</textarea>' : '';
 
-						if ( $ignore_global ) {
+						if ( $ignore_global && edac_is_pro() ) {
 							$html .= ( true === $ignore_permission ) ? '<a href="' . admin_url( 'admin.php?page=accessibility_checker_ignored&tab=global' ) . '" class="edac-details-rule-records-record-ignore-global">Manage Globally Ignored</a>' : '';
```
Comment (pattonwebz) [inline admin/class-ajax.php]: There's a translation function missing here.

```suggestion
							$html .= ( true === $ignore_permission ) ? '<a href="' . admin_url( 'admin.php?page=accessibility_checker_ignored&tab=global' ) . '" class="edac-details-rule-records-record-ignore-global">' . __( 'Manage Globally Ignored', 'accessibility-checker' ) . '</a>' : '';
```
